App.tsx

```diff
@@ -1,12 +1,48 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
 
 export default function App() {
-  return (
-    <View style={styles.container}>
-      <Text>Hello World!!!</Text>
-    </View>
-  );
+  const [vulpix, setVulpix] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  const getVulpix = async () => {
+    try {
+      const res = await fetch('https://pokeapi.co/api/v2/pokemon/37');
+      const data = await res.json();
```

Comment:
in the future it might be helpful to write custom type for `data`, something like this:
```ts
interface Pokemon {
  name: string;
  weight: number;
  height: number;
}
```
and then use it:
`const data: Pokemon = await res.json();`

Thanks to this TypeScript will suggest what you use after typing `vulpix.` and also "scream" when you try to access something which is not inside `Pokemon` interface